Assets/VRTK/Source/Scripts/Locomotion/VRTK_BasicTeleport.cs

```diff
@@ -219,7 +220,7 @@ protected virtual void OnEnable()
         {
             VRTK_PlayerObject.SetPlayerObject(gameObject, VRTK_PlayerObject.ObjectTypes.CameraRig);
             headset = VRTK_SharedMethods.AddCameraFade();
-            playArea = VRTK_DeviceFinder.PlayAreaTransform();
+            TransformToTeleport = VRTK_DeviceFinder.PlayAreaTransform();
```

Comment:
isn't this line always forcing it to be the play area?

Reply:
Yes I forgot to push my other changes, I will do that and change the casing.